RSConnectDIOToSnap7/Sharp7.cs

```diff
@@ -335,8 +335,10 @@ public void Close()
 
         private void CreateSocket()
         {
-            TCPSocket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
-            TCPSocket.NoDelay = true;
+            TCPSocket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp)
+            {
+                NoDelay = true
+            };

```

Comment:
Don't modify this file. It is official one.
At least, you can update it by the file you can found here:
https://sourceforge.net/projects/snap7/files/Sharp7/Last/1.0.6/